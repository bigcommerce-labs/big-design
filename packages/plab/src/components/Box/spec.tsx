import 'jest-styled-components';
import React from 'react';
import { render } from 'react-testing-library';

import { Box } from './index';

test('box should have margin props', () => {
  const { container, rerender } = render(<Box />);

  expect(container.firstChild).not.toHaveStyle('margin: 1rem');

  rerender(<Box margin="medium" />);

  expect(container.firstChild).toHaveStyle('margin: 1rem');
});
