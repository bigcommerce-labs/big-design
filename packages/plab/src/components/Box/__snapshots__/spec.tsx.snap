// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has border props 1`] = `
.c0 {
  border: 1px solid #D9DCE9;
}

<div
  class="c0"
/>
`;

exports[`has elevation props 1`] = `
.c0 {
  border-radius: 0.25rem;
  box-shadow: 0px 2px 12px rgba(49,52,64,0.2);
}

<div
  class="c0"
  elevation="floating"
/>
`;
